inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -1040,7 +1152,12 @@ linelist_cleaned %>%
 
 
 CFR by age group among inpatients with known outcomes
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+cfr_by_age_group
 
+This chunk gives the case fatality ratio among inpatients with outcomes, divided
+by age group. If you have no deaths reported, the table will not work well.
```

Comment:
Better to be "If you have no deaths reported, this table will not be useful." since the table will work without death:

``` r
library(tidyverse)
library(sitrep)
iris %>%
  mutate(tepals = FALSE) %>%
  case_fatality_rate_df(deaths = tepals, group = Species, mergeCI = TRUE)
#> # A tibble: 3 x 5
#>   Species    deaths population   cfr ci          
#>   <fct>       <int>      <int> <dbl> <chr>       
#> 1 setosa          0         50     0 (0.00--7.13)
#> 2 versicolor      0         50     0 (0.00--7.13)
#> 3 virginica       0         50     0 (0.00--7.13)
```

<sup>Created on 2019-09-02 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>